kubernetes/proposals/split-k8s-listener.md

```diff
@@ -0,0 +1,19 @@
+# This is a design proposal for changes in k8s-listener.
+
+Roadmap outlined by Juergen suggests we split k8s-listener.py into 2 standalone components.
+First component is to run on Kubernetess master node and dispatch orders to second component,
+running on every kube-node.
+
+* Proposed change is to replace `policy_update` call found in `process` function with
+`dispatch_orders` call that will retrieve ip addresses of all hosts from romana topology
+service, then loop over this addresses and send policy object with romana metadata to
+each host via  HTTP POST.
+* Introduce optional commadline parameter that would trigger `ListenAndServe` function in `main`
```

Comment:
So you envision to use the same code base, just turn the listener to a listener-agent in case that command line parameter is specified?

I guess that works. Please put comments into the various functions to indicate clearly which ones are the listener parts, and which the listener-agent parts.